docs/topics/http/views.txt

```diff
@@ -166,3 +166,39 @@ The :func:`~django.views.defaults.bad_request` view is overridden by
 
     Use the :setting:`CSRF_FAILURE_VIEW` setting to override the CSRF error
     view.
+
+Testing custom error views
+--------------------------
+
+To test the response of your custom error handler, you should raise an
+exception in your test view. Here is an example::
+
+    from django.core.exceptions import PermissionDenied
+    from django.http import HttpResponse
+    from django.test import SimpleTestCase, override_settings
+    from django.urls import path
+
+
+    def response_error_handler(request, exception=None):
+        return HttpResponse('Error handler content', status=403)
+
+
+    # Raise the exception in your view:
+    def permission_denied_view(request):
+        raise PermissionDenied
+
+
+    urlpatterns = [
+        path(r'^$', permission_denied_view),
+    ]
+
+    handler403 = response_error_handler
+
+
+    @override_settings(ROOT_URLCONF=__name__)
```

Comment:
I think it would be worthwhile to spell out that the `ROOT_URLCONF` override needs to be the same file that contains the `handler403` line.